actions-runner-controller.yaml

```diff
@@ -12,6 +12,8 @@ environment:
       - busybox
       - ca-certificates-bundle
       - go
+  environment:
```

Comment:
Did we spot an error where we needed to set this? the go/build wrapper should take care of setting this for FIPS vs non-fips now